phoenicis-repository/src/test/java/org/phoenicis/repository/repositoryTypes/CachedRepositoryTest.java

```diff
@@ -38,8 +38,8 @@ public void testFetchInstallableApplications() throws Exception {
                         .thenReturn(
                                 new RepositoryDTO.Builder()
                                         .withCategories(
-                                                Arrays.asList(new CategoryDTO.Builder().withName("Category 1").build(),
-                                                        new CategoryDTO.Builder().withName("Category 2").build()))
```

Comment:
That was also my first approach. However, it's not that simple. For example, you also have to consider every place where apps/categories are sorted and the search. You could not write code like the following any more:
```Java
Comparator.comparing(CategoryDTO::getName)
```
So you would have to be very careful when implementing something that you translate everything in the right places.
Therefore I decided that it's easier/cleaner to have the translated name in the DTO already.

In general I believe it's a good idea to identify apps/categories by an ID and not the name. That's also what Steam etc. does.

Additionally, we might have issues with names if they contain whitespace or special characters as @madoar pointed out above.